server/chat-plugins/helptickets.js

```diff
@@ -237,13 +276,65 @@ class HelpTicket extends Rooms.RoomGame {
 			const user = Users.get(ticketGameUser.userid);
 			if (user) user.updateSearch();
 		}
+		if (!this.involvedStaff.size) {
+			if (staff.isStaff && staff.userid !== this.ticket.userid) {
+				this.involvedStaff.add(staff.userid);
+			} else {
+				this.involvedStaff.add(toID(this.ticket.claimed));
+			}
+		}
+		this.writeStats(result);
+	}
+
+	/**
+	 * @param {(boolean | 'ticketban' | 'deleted')} result
```

Comment:
```suggestion
	 * @param {boolean | 'ticketban' | 'deleted'} result
```

don't mind me, just nitpicking